.github/workflows/github-actions-ci.yml

```diff
@@ -30,20 +30,23 @@ jobs:
         with:
           username: alleninstitutepika
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Determine github branch name
+        shell: bash
+        run: echo "GH_BRANCH_NAME=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_ENV
       - name: Build Docker Image
         run: |
           git clone https://github.com/AllenInstitute/ecephys_etl_pipelines.git
           cd ecephys_etl_pipelines
-          echo "Using branch $GITHUB_HEAD_REF"
-          git checkout $GITHUB_HEAD_REF
-          docker build --build-arg ECEPHYS_ETL_TAG=${GITHUB_HEAD_REF} --build-arg ECEPHYS_ETL_COMMIT_SHA=${GITHUB_SHA} --tag alleninstitutepika/ecephys_etl_pipelines:$GITHUB_SHA docker/
+          echo "Using branch ${{ env.GH_BRANCH_NAME }}"
+          git checkout "${{ env.GH_BRANCH_NAME }}"
+          docker build --build-arg ECEPHYS_ETL_TAG="${{ env.GH_BRANCH_NAME }}" --build-arg ECEPHYS_ETL_COMMIT_SHA=${GITHUB_SHA} --tag alleninstitutepika/ecephys_etl_pipelines:$GITHUB_SHA docker/
```

Comment:
this is not from this PR, but
```
--build-arg ECEPHYS_ETL_COMMIT_SHA=${GITHUB_SHA}
```
is not actually an arg in the docker file.

Reply:
https://github.com/AllenInstitute/ophys_etl_pipelines/blob/b73f30e044a39a00d075b297fa858eece14242b5/docker/Dockerfile#L11-L12

Reply:
I suggest adding it, as it is quite nice to have modules running through the docker file log something like:
`running this commit <SHA> of the repo`

Reply:
@djkapner Good catch, done.